### DISCUSSION TITLE
Questions about the intepretation -  integration of spatial datasets with single-cell data

### DISCUSSION THREAD

**Discussion by gmoralest** [satijalab/seurat#7059] [Q&A]
Hello, good morning, I am using the SpatialFeaturePlot() function to compare clusters obtained from a single cell analysis (DropSeq) against the same tissue analyzed using Spatial transcriptomics (10x Visium data).
For some of the clusters, the function tells me that it has not been found, and for others, it generates the following plot. I would like to know what is the difference in the interpretation of both situations.

![scRNAseq - Spatial_SAMPLE2_cluster_CRadial glial cells G2M 9](https://user-images.githubusercontent.com/85545966/226297467-32c29598-c0a2-46dd-9c6d-902d8a79e661.png)

 I would like to know what is the difference in the interpretation of both situations, specially when it generates the previous plot.

Thank you very much in advance

**Comment by dd315** [comment]
I have encountered the same problem, please help me

**Reply by baracus37** [reply]
I second this, please also see (https://github.com/satijalab/seurat/issues/8489) for reference.
